uui/components/tables/DataRowsContainer/DataRows.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useRef } from 'react';
+import { DataRowProps } from '@epam/uui-core';
+import css from './DataRowsContainer.module.scss';
+import { getChildrenAndRest } from './utils';
+
+interface GroupProps<TItem, TId> {
+    row: DataRowProps<TItem, TId>;
+    childrenRows: DataRowProps<TItem, TId>[];
+    renderRow: (props: DataRowProps<TItem, TId>) => React.ReactNode;
+    top: number;
+}
+
+export interface DataRowsProps<TItem, TId> {
+    rows: DataRowProps<TItem, TId>[],
+    renderRow: (props: DataRowProps<TItem, TId>) => React.ReactNode;
+    top: number;
+}
+
+function DataRowGroup<TItem, TId>({
+    row,
+    childrenRows,
+    renderRow,
+    top = 1,
+}: GroupProps<TItem, TId>) {
+    const rowRef = useRef<HTMLDivElement>();
+    const childrenPinnedTop = row.isPinned ? (top + (rowRef.current?.clientHeight ?? 0)) : top;
```

Comment:
As we'll get ref.current only after rendering, we won't get correct values for 'top' on the first rendering. However, this works, as we have few re-renders before some rows get stuck. Correct?

Reply:
Yes, that's correct.